docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -417,7 +423,31 @@
    "source": [
     "## Data Arrays\n",
     "\n",
-    "There is essentially only one way to construct [data arrays](../../generated/classes/scipp.DataArray.rst), namely its initializer:"
+    "A [DataArray](../../generated/classes/scipp.DataArray.rst)'s \"data\" is set to a [Variable](../../generated/classes/scipp.Variable.rst), and optionally a dict-like mapping, where the keys are the coordinates names and the values are coordinate variables.\n",
```

Comment:
This sentence is a bit strange. Did you mean to refer to the initializer?

Reply:
```suggestion
    "A [DataArray](../../generated/classes/scipp.DataArray.rst) is initialized from data given by a [Variable](../../generated/classes/scipp.Variable.rst), and optionally a dict-like mapping, where the keys are the coordinates names and the values are coordinate variables.\n",
```
like this maybe?